tests/test_units/test_fmuprovider_class.py

```diff
@@ -406,3 +407,19 @@ def test_fmuprovider_workflow_reference(fmurun_w_casemetadata, globalconfig2):
         pydantic.ValidationError, match="Input should be a valid string"
     ):
         FmuProvider(model=GLOBAL_CONFIG_MODEL, workflow=123.4).get_metadata()
+
+
+def test_ert_simulation_modes_one_to_one() -> None:
+    """Ensure dataio known modes match those defined by Ert. These are currently defined
+    in `ert.mode_definitions`. `MODULE_MODE` is skipped due to seemingly being relevant
+    to Ert internally -- the modes are duplicated there."""
```

Comment:
A comment here on the docstring style - I would like to enforce the "google standard", which is one
"punch line (in one line)", then a space before more explanation.

```
   """Ensure dataio known modes match those defined by Ert.

    These are currently defined in `ert.mode_definitions`. `MODULE_MODE` is skipped
    due to seemingly being relevant to Ert internally -- the modes are duplicated there.
    """
```

Reply:
It'd be nice to make this a linting check. Maybe we should have a team chat about doing so